Changes

```diff
@@ -1,6 +1,42 @@
 Release history for Zonemaster component Zonemaster-Backend
 
 
+v10.0.0 2022-12-19 (public release version)
+
+ [Deprecation]
+ - The use of language tags that includes country code is
+   deprecated. See "docs/API.md#language-tag". Such
+   language codes will be made illegal in the v2023.1 release.
+   Use language codes without country code instead.
+ - The use of an empty string in the "LANGUAGE.locale"
+   setting is deprecated and will be made illegal in the
+   v2023.1 release. See "docs/Configuration.md#language-section".
+ - The use of two or more locale tags with the same language code
+   in the configuration is deprecated and will be made illegal in the
+   v2023.1 release. See "docs/Configuration.md#language-section".
+
+ [Breaking changes]
+ - Removes database primary key from API method "get_test_results"
+   (#946, #949).
```

Comment:
#949 was included in last release so it should probably be removed from here.

Also it feels a bit silly to bump the major version just for this change. In future releases we should defer less important breaking changes until we've merged a breaking change that really matters.

Reply:
I know that #949 was in previous release. I just included it as reference.